flakeseries_table.go

```diff
@@ -0,0 +1,130 @@
+package gocassa
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/mattheath/base62"
+	"github.com/mattheath/kala/bigflake"
+	"github.com/mattheath/kala/util"
+)
+
+// we have to put the timestamp from the flake ID into a field so that we can
+// execute queries with it
+const flakeTimestampFieldName = "flake_created"
+
+type flakeSeriesT struct {
+	Table
+	bucketSize time.Duration
+}
+
+func (o *flakeSeriesT) Set(v interface{}) Op {
+	m, ok := toMap(v)
+	if !ok {
+		panic("Can't set: not able to convert")
+	}
+	id, ok := m["Id"].(string)
```

Comment:
The name of this field should probably be specified in the table constructor, as it is with the other tables.